nixos/modules/virtualisation/libvirtd.nix

```
@@ -145,6 +145,13 @@ in {
 
   config = mkIf cfg.enable {
 
+    assertions = [
+      {
+        assertion = config.security.polkit.enable;
+        message = "The libvirtd module requires Polkit to be enabled ('security.polkit.enable = true').";
```

What if I change it like this?
```suggestion
        message = "The libvirtd module currently requires Polkit to be enabled ('security.polkit.enable = true').";
```

---

I'm not sure, it should be possible to setup an alternative method currently via `extraConfig`, shouldn't it?

---

I'm not sure, it seems it always sets `auth_unix_ro` and `auth_unix_rw` to `polkit`, regardless of what extraConfig is set to
```nix
  configFile = pkgs.writeText "libvirtd.conf" ''
    auth_unix_ro = "polkit"
    auth_unix_rw = "polkit"
    ${cfg.extraConfig}
  '';
```
Unless later options override earlier ones

---

Ah sorry, missed that. Then your change seems sensible indeed.